less/flipcard.less

```diff
@@ -159,7 +166,7 @@
           // desktop
           padding: 20px 60px;
         }
-
+        
         .ie8 &,
```

Comment:
I know it's outside the scope of the task, but IE8/9 styling can be removed here

Reply:
I'm happy either way here, as we are (going to be) tracking all plugins legacy code and refactor on https://learningpool.atlassian.net/browse/ADAPT-5202 and might be better to keep this PR simple.

Reply:
Will leave it and add to the epic for plugins re-factor. Then do it all together.